app/controllers/organizations/staff/dashboard_controller.rb

```diff
@@ -12,40 +14,20 @@ def index
   end
 
   def incomplete_tasks
```

Comment:
A thought here - we are returning a collection of pets. But we are conflating them with tasks. Would it be more accurate to change the method names to `pets_with_incomplete_tasks` and `pets_with_overdue_tasks` and change the partial name to `pets` instead of `tasks`?

Reply:
hmm.. in that case we need to change the routes as well.